src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] it would be better if the App component classed based not function tin order to match the requirements 
-Screenshot 
<img width="767" alt="Screen Shot 2022-10-26 at 15 50 55" src="https://user-images.githubusercontent.com/61790587/198059486-997a655f-558d-464e-815b-899cecdf1e62.png">